jib-core/src/test/java/com/google/cloud/tools/jib/builder/BuildConfigurationTest.java

```diff
@@ -153,4 +153,14 @@ public void testBuilder_missingValues() {
           ex.getMessage());
     }
   }
+
+  @Test
+  public void testValidJavaClassRegex() {
+    Assert.assertTrue("my.Class".matches(BuildConfiguration.VALID_JAVA_CLASS_REGEX));
+    Assert.assertTrue("my.java_Class$valid".matches(BuildConfiguration.VALID_JAVA_CLASS_REGEX));
+    Assert.assertFalse("${start-class}".matches(BuildConfiguration.VALID_JAVA_CLASS_REGEX));
```

Comment:
can you verify that `is123.valid` works?
and that mutliple package items work? `abc.xyz.Valid`